TestProjects/HDRP_Tests/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -815,4 +818,4 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/GraphicTests/Scenes/9x_Other/9910_GlobalMipBias.unity
     guid: 3f9d18568d46df044847954b624b6152
-
+  m_configObjects: {}
```

Comment:
side note: last time I see someone touching this it cause some failure on Yamato, so let's monitor yamato to be sure it is green

Reply:
I can remove this change since we probably don't need it